src/Core/SettingsChangesHistory.cpp

```diff
@@ -81,6 +81,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"restore_replicated_merge_tree_to_shared_merge_tree", false, false, "New setting."},
             {"parallel_replicas_only_with_analyzer", false, true, "Parallel replicas is supported only with analyzer enabled"},
             {"s3_allow_multipart_copy", true, true, "New setting."},
+            {"save_variant_with_escaped_filename", true, true, "New setting."},
```

Comment:
```suggestion
            {"save_variant_with_escaped_filename", false, true, "New setting."},
```
We must keep old value as `false` for compatibility.